src/main.cpp

```diff
@@ -166,34 +196,77 @@ void loop()
   char tempString[10];
   dtostrf(temperature, 1, 2, tempString);
 
-  Serial.print("Temperature: ");
-  Serial.print(tempString);
-  Serial.println(" °C");
+  // Serial.print("Temperature: ");
+  // Serial.print(tempString);
+  // Serial.println(" °C");
 
   // Send temperature over BLE
   pTempCharacteristic->setValue(tempString);
-  pTempCharacteristic->notify();
+  // Don't need to notify so often
+  // pTempCharacteristic->notify();
+
+  // Get scale reading
+  float weight = scale.get_units(1);
```

Comment:
I changed this from 10 -> 1. 10 meant it was giving me the average of 10 readings. We just want the raw data and handle the averaging ourselves